llvm/test/CodeGen/AArch64/GlobalISel/select.mir

```diff
@@ -183,6 +183,14 @@ registers:
   - { id: 5, class: gpr }
   - { id: 6, class: gpr }
   - { id: 7, class: gpr }
+  - { id: 8, class: fpr }
```

Comment:
As a separate cleanup this test should drop the explicit registers section and use the inline register class syntax

Reply:
Sure, will do.

Reply:
See #87013